aten/src/TH/generic/THTensor.cpp

```diff
@@ -596,6 +596,20 @@ int THTensor_(isContiguous)(const THTensor *self)
   return 1;
 }
 
+int THTensor_(isSize)(const THTensor *self, const THLongStorage *dims)
```

Comment:
why is this here?  It was removed in https://github.com/pytorch/pytorch/pull/10146 -- your merge probably went awry.